numpy/lib/function_base.py

```diff
@@ -764,18 +764,20 @@ def copy(a, order='K', subok=False):
     >>> x[0] == z[0]
     False
 
-    Note that, NumPy will not copy the elements of object arrays:
+    Note that, ``np.copy`` will not copy the elements of object
```

Comment:
```suggestion
    Note that ``np.copy`` will not copy the elements of object
```
This comma doesn't make sense here.